### DISCUSSION TITLE
how to caculate AFM  system by vasp+wannier90

### DISCUSSION THREAD

**Discussion by 1638927697** [mailhexu/TB2J#13] [Q&A]
I encountered difficulties in caculating AFM materials by vasp+wannier90 . You mentioned that there is examples about vasp in ./examples , but I could't find it . Can you provide me with those sample files ? My email is 22036013@zju.edu.cn .Please forgive my poor English.

**Comment by mailhexu** [comment]
See https://github.com/mailhexu/TB2J_examples/tree/master/Wannier

Best wishes,
HeXu

**Reply by 1638927697** [reply]
Thanks for your answer sincerely.Now I want to caculate  the biquadratic exchange of materials ,but I could't understand how to caculate it after I read the manual .Or  dose it must use TB2J-SIESTA ? Can you tell me where can I find relevant information?

**Comment by mailhexu** [comment]
Hello,
The calculation of the biquadratic term is being removed from TB2J. In some
cases from my test, the biquadratic exchange I get from the formalism
implemented in TB2J doesn't look quite right.
You can do a fitting of Jij(theta), where theta is the angle between the
two magnetic atoms i and j, to get the bilinear and biquadratic exchanges.
Best regards,
HeXu
